unpacked/regular/browser_dragdrop_incompat/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "manifest_version": 2,
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "dragdrop-incompat@tests.mozilla.org",
+      "strict_max_version": "45.0"
```

Comment:
Any idea why we have that?

Reply:
> Any idea why we have that?

it looks like we are doing it as part of a test case that is explicitly covering expected behavior on trying to install an incompatible addon: https://searchfox.org/mozilla-central/rev/0e9ea50a999420d93df0e4e27094952af48dd3b8/toolkit/mozapps/extensions/test/browser/browser_dragdrop.js#257-270

The previous version of the test was also hitting the same scenario with a test extension with an incompatible version range specified in an install.rdf file: https://hg.mozilla.org/mozilla-central/rev/d854ebbcbb7e39b29508c856b118ca52f6623d53#l13.22

Honestly I think we could test this scenario without having to use a signed xpi, but to simulate it with the cert db override trick we often use in our tests (I guess we may have introduced the cert db approach after this test extension got ported from install.rdf to manifest.json).

I think that reducing the number of signed xpi we use in mozilla-central tests to the minimum would be great, and so it would be good to followup later, but in the short term may be worth keeping this manifest as is, wdyt?

Reply:
I second this. Assuming we can rewrite the test, let's not sign that one.